tools/clang/test/HLSLFileCheck/pix/DebugAsGroupSharedTrickyTypesPayload.hlsl

```diff
@@ -0,0 +1,28 @@
+// RUN: %dxc -enable-16bit-types -Od -Emain -Tas_6_6 %s | %opt -S -hlsl-dxil-PIX-add-tid-to-as-payload,dispatchArgY=3,dispatchArgZ=7 | %FileCheck %s
```

Comment:
Don't see why enable-16bit-types is needed. Harmless though.

Reply:
Just to exercise the "half" properly.

Reply:
Oh right, this file has no halfs/(halves?)! That's a copy-paste from the other file that does have one half. A whole half, that is.